azure-kusto-ingest/azure/kusto/ingest/kusto_ingest_client.py

```diff
@@ -79,12 +79,7 @@ def __init__(self,
                                          username=username,
                                          password=password,
                                          authority=authority)
-        self._last_time_refreshed_containers = datetime.min
-        self._temp_storage_objects = None
-        self._last_time_refreshed_queues = datetime.min
-        self._queues = None
-        self._last_time_refreshed_token = datetime.min
-        self._kusto_token = None
+        self._resources_manager = _ResourceManager(self._kusto_client)
```

Comment:
Do we need self._kusto_client as a memeber?

Reply:
Yes, this is a little weird relationship: client -> resourceManager -> client
And the name should be resource_manager

Reply:
You are right, we don't need him as a member in the ingest-client.
So overall the relationship we have is: ingest-client -> resourceManager -> client.